free-2.md

```diff
@@ -11,41 +11,82 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Currently, the competetive Pokémon meta has grown very stale, with the same
+pokémon on each team. I think that this is partially due to the complexity of
+differnt builds and values leading to such a huge variety of thigns that might
+work, that players find it hard to find new viable builds. In addition, testing
+new theories is a very time consuming process. Hopefully, this could speed up
+testing.
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A language to specify the moves and values of a pokémon could be very useful as
+it would allow the user to see what the effect would be by replacing a move or
+redistributing Effort Values.
+
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I came up with this idea mostly because I had been thinking about Pokémon a
+lot due to [proposal 1](free-1.md).
+
 
 ### Domain
 _Describe the project's domain in five words._
 
+Simulating battles by various pokémon.
+
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+The user would want to fully specify each of the 6 Pokémon (e.g. moves, EVs,
+IVs, Abilities) in their team as well as the structure of the simulation (e.g.
+10 different opponents, each fought 10 times).
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+The interpreter would read the team, then simulate various battles against
+random opponents. Maybe the user could also specify a team to play against.
```

Comment:
This seems hard because you'd need to somehow encode "best" moves into a game; it turns into a decision making problem here, and I suspect would end up missing valid strategies that still work.